### ISSUE TITLE
onEnd won't run if the handler has child nodes covering it

### ISSUE THREAD

**Issue by svntax** [knadh/dragmove.js#6]
On line 53,

```javascript
if (onEnd && e.target === handler) {
  onEnd(target, parseInt(target.style.left), parseInt(target.style.top));
}
```

If the handler has child nodes covering it, `e.target` will end up being the child node you clicked on, not the handler itself, so onEnd won't run. One way to replicate this bug is through the example in docs/index.html by changing it like so:

Add a span element to one of the handlers:
```html
<div id="test" class="parent" data-drag-bound="true">
  <div class="handle" style=""><span>Drag here</span></div>
</div>
```

and make sure .handle has a minimum height, like say 24px for this example:
```css
.handle {
  background: #888;
  cursor: move;
  width: 130px;
  min-height: 24px;
  color: #fff;
  padding: 10px;
}
```

Then try dragging the element outside the view by clicking on its text first, and you'll see that the snapping won't happen.

I was working on adding buttons and menus in the handler nodes in my project, which is how I found the bug. The dragging works perfectly fine, just to be clear. It's just onEnd that's the issue.

**Comment by knadh** [comment]
The mouse events are bound on `document` and not the handler, so it's not possible to use `event.currentTarget` to ignore the children and get the handler directly.

The right way to control this externally is to apply [`pointer-events: none`](https://developer.mozilla.org/en-US/docs/Web/CSS/pointer-events) (CSS) to the child elements inside your handler.

**Comment by knadh** [comment]
There was a much simpler (duh!) fix for this https://github.com/knadh/dragmove.js/pull/7. It's released in `v0.1.2`.
